### PR TITLE
Test

### PR DESCRIPTION
**Pull request by Teiko19**
Add character list
image

### CODE REVIEW

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    fetchData()
+      .catch(console.error);;
+  }, []);
+
+  return (
+    <header className="">
+      <div className="row row-cols-1 row-cols-sm-2 row-cols-md-3 g-3">
+        {
+          characters.map((x) => (
+            <div className="border border-black rounded bg-black text-white">
```
Comment (emibel) [inline src/components/Header/Header.tsx]: extract this into a new component

File: src/components/navBar/navBar.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+export const NavBar = () => {
+  return (
+      <nav className="navBar navbar navbar-expand-lg navbar-light bg-light">
+        <div>
+          <a className="navbar-brand" href="#">Home</a>
+        </div>
+
+        <div className="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul className="navbar-nav mr-auto">
```
Comment (emibel) [inline src/components/navBar/navBar.tsx:11]: iterate an array to create this list

File: src/components/Header/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const json = await fetch(API_CHARACTERS);
+      const data = await json.json();
+      setCharacters(data.results);
+    }
+    fetchData()
+      .catch(console.error);;
+  }, []);
```
Comment (emibel) [inline src/components/Header/Header.tsx]: excelent

**Comment by emibel** [comment]
* build has failed
* missing images in the PR description

File: src/components/character-list/character-list.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+      const json = await fetch(API_CHARACTERS);
+      const data = await json.json();
+      setCharacters(data.results);
+    }
+    fetchData()
+      .catch(console.error);
+  }, []);
+
+  return (
+    <header className="">
```
Comment (emibel) [inline src/components/character-list/character-list.tsx]: remove header

File: src/components/character-list/character-list.tsx [hunk 5]
```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect } from "react";
+import styles from './Header.module.scss';
```
Comment (emibel) [inline src/components/character-list/character-list.tsx]: remove scss file

File: src/components/navBar/navBar.tsx [hunk 6]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import './navBar.module.scss'
```
Comment (emibel) [inline src/components/navBar/navBar.tsx]: import styles from './navBAr....'

Comment (emibel) [reply src/components/navBar/navBar.tsx]: change class name to navbar-container

File: src/components/character-list/character.tsx [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+        <ul className="list-group">
+          <li className="list-group-item"><p><b>Gender:</b> {character.gender} </p></li>
+          <li className="list-group-item"><p><b>Status:</b> {character.status} </p></li>
+          <li className="list-group-item"><p><b>Species:</b> {character.species} </p></li>
+          <li className="list-group-item"><p><b>Origin:</b> {character.origin.name} </p></li>
+          <li className="list-group-item"><p><b>Location:</b> {character.location.name} </p></li>
+        </ul>
+      </div>
+      <div className="d-flex justify-content-center mb-1 mt-1">
+        <img src={character.image} className="img-responsive rounded-circle" alt="Responsive image"></img>
```
Comment (emifsl) [inline src/components/character-list/character.tsx]: Use `<Image />` from `next/image`

File: src/components/character-list/character-list.tsx [hunk 8]
```diff
@@ -0,0 +1,30 @@
+import React, { useState, useEffect } from "react";
+import { API_CHARACTERS } from "../../API/constant";
```
Comment (emifsl) [inline src/components/character-list/character-list.tsx]: import { API_CHARACTERS } from "../../api/constant";

File: src/components/navBar/navBar.module.scss [hunk 9]
```diff
@@ -0,0 +1,5 @@
+.navBar_container {
+    background-color: black;
```
Comment (emifsl) [inline src/components/navBar/navBar.module.scss]: use SCSS  variable for this color

File: src/components/character-list/character-list.tsx [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+      const json = await fetch(API_CHARACTERS);
+      const data = await json.json();
+      setCharacters(data.results);
+    }
+    fetchData()
+      .catch(console.error);
+  }, []);
+
+  return (
+    <div>
```
Comment (emifsl) [inline src/components/character-list/character-list.tsx]: you dont need this div

**Review by emibel — CHANGES_REQUESTED** [review]
pls fix comments

File: src/components/character-list/character.tsx [hunk 11]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import Image from "next/image";
+
+export const Character = ({ character }) => {
+  return (
+    <div className="border border-black rounded bg-black text-white">
+      <div className="">
```
Comment (emibel) [inline src/components/character-list/character.tsx]: remove className is it has no classes

**Review by emibel — CHANGES_REQUESTED** [review]
fix comments

**Comment by emibel** [comment]
Vercel — Deployment has failed

File: src/components/character-list/character-list.tsx [hunk 12]
```diff
@@ -1,5 +1,5 @@
 import React, { useState, useEffect } from "react";
-import { API_CHARACTERS } from "../../api/constant";
+import { API_CHARACTERS } from "../api/constant";
 import 'bootstrap/dist/css/bootstrap.css'
```
Comment (emibel) [inline src/components/character-list/character-list.tsx]: remove this from here...


this import applies to the entire app

**Review by emibel — APPROVED** [review]

**Pull request merged**